assets/src/components/admin/admin_tables.tsx

```diff
@@ -204,14 +206,21 @@ const BusEinkV2ScreensTable = (): JSX.Element => {
     return app_id === "bus_eink_v2";
   };
 
-  const columns = [...v2Columns, departuresColumn, footerColumn, alertsColumn];
+  const columns = [
+    ...v2Columns,
+    departuresColumn,
+    footerColumn,
+    alertsColumn,
+    headerColumn,
```

Comment:
Maybe the header column should be first here (after `v2Columns`), since it was earlier in the order prior to the split. Though we are still unavoidably moving things around a bit (it used to go Header => Evergreen => ...).

Reply:
updated this ordering!